README.md

```diff
@@ -124,6 +123,12 @@ Vertical or horizontal lines are supported.
 
 	// Mouse event handlers - be sure to enable the corresponding events in the
 	// annotation events array or the event handler will not be called.
+	// See https://developer.mozilla.org/en-US/docs/Web/Events for a list of
+	// supported mouse events.
+	onMouseenter: function(e) {},
```

Comment:
Eventually, yes, but it will need some care to do it right in a way that works cross-browser and also works for the user. Probably need some hit radius logic added, and integration with Hammer.js. That's more than I can get into right now.